bin/crail

```diff
@@ -78,4 +77,4 @@ if [ -f "${CONF_PATH}/crail-env.sh" ]; then
   set +a
 fi
 
-exec "$JAVA" -Dproc_$COMMAND -Dsun.nio.PageAlignDirectMemory=true $CRAIL_EXTRA_JAVA_OPTIONS $CLASS "$@"
+exec "java" -Dproc_$COMMAND -Dsun.nio.PageAlignDirectMemory=true $CRAIL_EXTRA_JAVA_OPTIONS $CLASS "$@"
```

Comment:
We want the user to be able to switch between JVMs easily. The JAVA_HOME environment variable is a pretty common way to do this. However, I understand that this might be a hassle, so we could introduce a check if JAVA_HOME is set and if it is not set set JAVA to "java".